TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -329,6 +335,68 @@ public static void ScanTokensLineContinuationGroup(IList<TokensLine> continuatio
                     // Check if the last token so far is an alphanumeric or national literal
                     if (lastTokenOfConcatenatedLineSoFar.TokenFamily == TokenFamily.AlphanumericLiteral)
                     {
+                        if (!lastTokenOfConcatenatedLineSoFar.HasClosingDelimiter)
```

Comment:
You may define some constants for both `'` and `"` to improve readability in this portion of code.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1604/commits/5de0690f865a31192983ee6452e96a892455d316